core/src/test/java/cucumber/runtime/formatter/HTMLFormatterTest.java

```diff
@@ -61,15 +58,38 @@ public void writes_index_html() throws Throwable {
     @Test
     public void writes_valid_report_js() throws Throwable {
         writeReport();
-        URL reportJs = new URL(outputDir, "report.js");
-        Context cx = Context.enter();
-        Global scope = new Global(cx);
-        try {
-            cx.evaluateReader(scope, new InputStreamReader(reportJs.openStream(), "UTF-8"), reportJs.getFile(), 1, null);
-            fail("Should have failed");
-        } catch (EcmaError expected) {
-            assertTrue(expected.getMessage().startsWith("ReferenceError: \"document\" is not defined."));
-        }
+        String reportJs = FixJava.readReader(new InputStreamReader(new URL(outputDir, "report.js").openStream(), "UTF-8"));
+        assertEquals("$(document).ready(function() {var formatter = new CucumberHTML.DOMFormatter($('.cucumber-report'));formatter.uri(\"some\\\\windows\\\\path\\\\some.feature\");\n" +
```

Comment:
@mlvandijk @mpkorstanje assertEquals cannot be used to compare things with json data structures in them, since json data structures should be considered equal also if the attribute order differs. Thus we got a build failure when this PR was merged: https://travis-ci.org/cucumber/cucumber-jvm/jobs/273568042.
It is now fixed in https://github.com/cucumber/cucumber-jvm/commit/fb1595232f4a7fddfde33c9c3c63928fd906707b